frontend/src/views/Game.tsx

```diff
@@ -10,20 +16,45 @@ type LocationState = {
 function GamePage() {
   const location = useLocation<LocationState>();
   const [room, setRoom] = useState<Room | null>(null);
+  const [problems, setProblems] = useState<Problem[] | null>(null);
+  const [error, setError] = useState<string | null>(null);
 
   // Called every time location changes
   useEffect(() => {
     if (location && location.state && location.state.room) {
       setRoom(location.state.room);
     }
+    getProblems().then((res) => {
+      if (isError(res)) {
+        setError((res as ErrorResponse).message);
+        setProblems(null);
+      } else {
+        setProblems(res as Problem[]);
+        setError(null);
+      }
+    });
   }, [location]);
 
+  const firstProblem = problems?.[0];
+
   return (
     <div>
-      Room:
-      {' '}
-      {room ? room.roomId : 'No room joined'}
-      <Editor height="90vh" language="javascript" />
+      <FlexContainer>
+        <Header />
+        <FlexInfoBar>
+          Room:
+          {' '}
+          {room ? room.roomId : 'No room joined'}
+        </FlexInfoBar>
+        <FlexPanel>
+          <ProblemHeaderText>{ firstProblem?.name }</ProblemHeaderText>
+          <Text>{ firstProblem?.description }</Text>
+          <ErrorText>{error ? <ErrorMessage message={error} /> : null}</ErrorText>
+        </FlexPanel>
+        <FlexPanel>
+          <Editor height="100vh" language="javascript" />
```

Comment:
Not sure if the full-height thing has been fixed, but if not, I think it may have to do with the `100vh` here. Not sure if the solution would be to get rid of it (making it default back to `100%`) or something else though.

Reply:
Yep, it got fixed --  it was actually changing % to vh that did it haha